### PR TITLE
[WIP] Feature/compatibility zfc rbac next version

### PR DESCRIPTION
**Pull request by basz**
WIP!
Not sure if this will work... Let's see what travis says

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,15 +20,21 @@
...
         "prooph/service-bus": ">=4.3 <6.0",
-        "zf-commons/zfc-rbac": "^2.5.2",
-        "zfr/rbac": "^1.2"
+        "zf-commons/zfc-rbac": "dev-WIP-remove-identity-provider as 3.0 || ^2.5.2 || ^3.0"
     },
     "require-dev": {
+        "zendframework/zend-authentication": "~2.5",
         "phpunit/phpunit": "~4.8",
-        "fabpot/php-cs-fixer": "1.7.*",
+        "friendsofphp/php-cs-fixer": "1.7.*",
```
Comment (prolic) [inline composer.json]: ^1.11.5

**Comment by basz** [comment]
@prolic second try. last commit updates to use php7, phpunit, docheader, travis notifications, return types. I don't believe to is possible to support service bus 6 plus earlier versions. That would legitimise a v2 release. It should work with v2 and v3 of the zfc-rbac

**Comment by prolic** [comment]
@basz a new major release of service-bus-zfc-rbac-bridge is possible. I just don't have the resources to work on this at the moment besides helping out with code reviews.

**Comment by basz** [comment]
hm, I'm not using this package anymore. I see about ~10 install per day via packagist, so i'll think i'l wait a bit with updating this.

**Comment by basz** [comment]
which is weird now i think of it as i do use zfc-rbac and the service-bus 😕

**Pull request closed without merging**